tests/baselines/reference/quickInfoDisplayPartsInternalModuleAlias.baseline

```diff
@@ -38,7 +38,15 @@
           "kind": "space"
         },
         {
-          "text": "a1",
```

Comment:
This corresponds to this line

https://github.com/microsoft/TypeScript/blob/49533168dbb4e19f243b9dbdfd6a3aac69f5b3dd/tests/cases/fourslash/quickInfoDisplayPartsInternalModuleAlias.ts#L8

which seems like - I don't know, it feels like a regression that this doesn't display `a1` at all anymore.